src/main/java/uk/nhs/digital/nhsconnect/nhais/service/RegistrationConsumerService.java

```diff
@@ -73,22 +74,17 @@ public void handleRegistration(MeshMessage meshMessage) {
             .collect(Collectors.toList());
     }
 
-    private MeshMessage prepareRecepOutboundMessage(Interchange interchange) {
-        //TODO: NIAD-390
-//        var recepMeshMessage = buildRecepMeshMessage(recep);
-//        LOGGER.debug("Wrapped recep in mesh message: {}", recepMeshMessage);
-//        outboundMeshService.publishToOutboundQueue(recepMeshMessage);
-//        LOGGER.debug("Published recep to outbound queue");
-        return null;
+    private OutboundState prepareRecepOutboundState(Interchange recep) {
+        if (recep.getMessages().size() != 1) {
+            throw new EdifactValidationException("Recep should have a 1 message");
```

Comment:
This is not necessary since we are building the recep and we know we are creating 1 message inside but it's not bad to check again